js/charts/chorddiagram.js

```diff
@@ -386,17 +386,23 @@ function chorddiagram() {
             .style('font-style', _fontStyle)
             .style('font-weight', _fontWeight)
             .text(function (d) {
-                var h = _pythagorousTheorem(r * Math.cos(angle(d)), r * Math.sin(angle(d)));
+                if (!_print) {
+                    var h = _pythagorousTheorem(r * Math.cos(angle(d)), r * Math.sin(angle(d)));
 
-                var a = angle(d);
-                var textAnchor = a < π2 || a > τ - π2 ? "start" : "end";
-                if (textAnchor == "start") {
-                    size = width / 2 - outerRadius * (r * Math.cos(angle(d)) / h) * 1.05
+                    var a = angle(d);
+                    var textAnchor = a < π2 || a > τ - π2 ? "start" : "end";
+                    if (textAnchor == "start") {
+                        size = width / 2 - outerRadius * (r * Math.cos(angle(d)) / h) * 1.05
```

Comment:
create separate function for below lines so that i can tested in future separatly 

width / 2 - outerRadius * (r * Math.cos(angle(d)) / h) * 1.05